core/src/commonMain/kotlin/entity/Attachment.kt

```diff
@@ -58,12 +60,25 @@ public data class Attachment(val data: AttachmentData, override val kord: Kord)
      */
     val width: Int? get() = data.width.value
 
+    /**
+     * The duration of the audio file (currently for voice messages).
+     */
+    val duration: Duration? get() = data.durationSecs.value
+
+    /**
+     * A sampled waveform (currently for voice messages).
+     */
+    val waveForm: ByteArray? by lazy {
+        data.waveform.value?.decodeBase64Bytes()
+    }
```

Comment:
```suggestion
    val waveForm: ByteArray? get() = data.waveform.value?.decodeBase64Bytes()
```
probably not worth caching, it's rather short + mutable